src/main/java/com/blogify/blogapi/repository/CommentReactionRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.blogify.blogapi.repository;
+
+import com.blogify.blogapi.model.CommentReaction;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface CommentReactionRepository extends JpaRepository<CommentReaction, Long> {
```

Comment:
, String> NOT , Long>

Reply:
because the type of id in CommentReaction is String